cfme/tests/automate/test_vmware_methods.py

```diff
@@ -23,7 +23,7 @@
 
 def pytest_generate_tests(metafunc):
     argnames, argvalues, idlist = testgen.provider_by_type(
-        metafunc, ['virtualcenter'], 'provisioning')
+        metafunc, ['virtualcenter'], required_fields=['provisioning'])
```

Comment:
Here, we require just `'provisioning'` instead of `'provisioning', 'templates'` because we assume that having a provisioning field without `'template'` in it would have no sense, right?